docs/advanced/localization.md

```diff
@@ -106,3 +106,23 @@ xpath = "//*[text()='{L10N:HomePage.welcomeText}'"
 # actual value at run-time:
 xpath = "//*[text()='Willkommen bei Wikipedia'"
 ```
+## Creation of multi-language tests
+If one or a group of tests checks several language versions of the site, then you need to do the following:
+1. (Optional) if you need to change the language version of the site, you can overwrite the locale parameter:
```

Comment:
optional usually is written in square brackets: `[Optional]`

Reply:
done